tensorflow/python/tools/optimize_for_inference_test.py

```diff
@@ -28,6 +28,7 @@
 from tensorflow.python.framework import test_util
 from tensorflow.python.ops import array_ops
 from tensorflow.python.ops import gen_nn_ops
+from tensorflow.python.ops import gen_math_ops
```

Comment:
The test is failing because of a dependency error:
```
ERROR: Strict deps violations: //tensorflow/python/tools:optimize_for_inference_test
  In tensorflow/python/tools/optimize_for_inference_test.py, no direct deps found for imports:
    line 31: from tensorflow.python.ops import gen_math_ops: Module "tensorflow.python.ops" not provided by a direct dep
```

Reply:
I probably use a different build environment, this failure didn't happen to my local runs, all the test cases passed. Anyway, I added a dependency in the BUILD file and hopefully, this will fix the error. 
Thanks again for helping with this old PR.

Reply:
Our internal tooling is more strict about build dependencies.  Will run it again.